definitions/contested/json/ComplexTypes/DFR-3516-nonprod.json

```diff
@@ -0,0 +1,95 @@
+[
+  {
+    "LiveFrom": "17/01/2025",
+    "ID": "FR_attachmentsToShare",
+    "ListElementCode": "documentId",
+    "FieldType": "Text",
+    "ElementLabel": " ",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "17/01/2025",
+    "ID": "FR_attachmentsToShare",
+    "ListElementCode": "attachmentName",
+    "FieldType": "Text",
+    "ElementLabel": " ",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "17/01/2025",
+    "ID": "FR_attachmentsToShare",
+    "ListElementCode": "documentToShare",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Send?",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "17/01/2025",
+    "ID": "FR_orderToShare",
+    "ListElementCode": "documentId",
+    "FieldType": "Text",
+    "ElementLabel": " ",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "17/01/2025",
+    "ID": "FR_orderToShare",
+    "ListElementCode": "documentName",
+    "FieldType": "Text",
+    "ElementLabel": " ",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "17/01/2025",
+    "ID": "FR_orderToShare",
+    "ListElementCode": "documentToShare",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Send?",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "17/01/2025",
+    "ID": "FR_orderToShare",
+    "ListElementCode": "hasSupportingDocuments",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "<hidden field>",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "17/01/2025",
+    "ID": "FR_orderToShare",
+    "ListElementCode": "includeSupportingDocument",
+    "FieldType": "MultiSelectList",
+    "FieldTypeParameter": "FR_yesOnly",
+    "ElementLabel": "Include supporting documents",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "17/01/2025",
+    "ID": "FR_orderToShare",
+    "ListElementCode": "attachmentsToShare",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "FR_attachmentsToShare",
+    "ElementLabel": " ",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "17/01/2025",
+    "ID": "FR_ordersToSend",
+    "ListElementCode": "value",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "FR_orderToShare",
+    "ElementLabel": "Which order(s) would you like to send?",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  }
```

Comment:
Just checking that you want ** FR_ordersToSend** as the ones above are FR_orderToSend (no s).

Reply:
Ah, seen the event id elsewhere.